src/places/places.controller.ts

```diff
@@ -87,7 +89,21 @@ export class PlacesController {
         return await this.demandsService.getDetailedDemandsForPlace(transaction, id);
       });
     } catch (error) {
-      throw new HttpException('Cannot get demands for place', HttpStatus.BAD_REQUEST);
+      throw new HttpException(`Cannot get demands for place ${id}`, HttpStatus.BAD_REQUEST);
+    }
+  }
+
+  @SetMetadata(MetadataKey.ALLOWED_ROLES, [UserRole.ADMIN, UserRole.PLACE_MANAGER])
+  @UseGuards(AuthGuard)
+  @HttpCode(HttpStatus.NO_CONTENT)
```

Comment:
I think you could also add swagger decorator to document the response if it is not documented properly

Reply:
Looks like it's documented properly based on `@HttpCode`